packages/cli/src/commands/test.ts

```diff
@@ -68,6 +66,11 @@ export default class Test extends AuthCommand {
       description: 'Always show the logs of the checks.',
       allowNo: true,
     }),
+    reporter: Flags.string({
+      char: 'r',
+      description: 'A list of custom reporters for the test output.',
+      default: 'list',
```

Comment:
You can actually pass here the valid reporters options as well

Reply:
Done.